scripts/_common.sh

```diff
@@ -39,7 +39,7 @@ ynh_install_composer () {
 	# Manage arguments with getopts
 	ynh_handle_getopts_args "$@"
 	workdir="${workdir:-$final_path}"
-	phpversion="${phpversion:-7.0}"
+	phpversion="${phpversion:-7.3}"
```

Comment:
This will now be handled by [this ugly hack in the core](https://github.com/YunoHost/yunohost/blob/buster-unstable/src/yunohost/app.py#L2809)

So for backward compatibility (installs in stretch) we should keep this line as using 7.0 by default ( - or if we don't care about backward compatibility, we should require yunohost >= 4.0 in the manifest)